ci-operator/step-registry/stackrox/install-helm/stackrox-install-helm-commands.sh

```diff
@@ -247,3 +293,21 @@ done
 # Cleanup nohup
 kill -9 "$(cat "${SCRATCH}/port_forward_pid")"
 rm "${SCRATCH}/port_forward_pid"
+
+if [[ "${ROX_SCANNER_V4}" == "true" ]]; then
+  echo ">>> Wait for 'stackrox scanner-v4' deployments"
+  wait_deploy scanner-v4-db
+  wait_deploy scanner-v4-indexer
+  if [[ -n "${SCANNER_V4_MATCHER_READINESS}" ]]; then
+    echo '>>> Follow scanner-v4-matcher logs until ready state'
+    kubectl wait pods --for=condition=Ready --selector 'app=scanner-v4-matcher' -n stackrox \
+      --timeout="${SCANNER_V4_MATCHER_READINESS_MAX_WAIT}" \
+      || { kubectl logs --tail=20 --selector 'app=scanner-v4-matcher' -n stackrox --timestamps; exit 1; }
+    kubectl rollout status deploy/scanner-v4-matcher --timeout=0 -n stackrox
+  else
+    wait_deploy scanner-v4-matcher "${SCANNER_V4_MATCHER_READINESS_MAX_WAIT}"
+  fi
```

Comment:
Related to: `wait_deploy scanner-v4-matcher "${SCANNER_V4_MATCHER_READINESS_MAX_WAIT}"` - I think we are not using 2nd argument in `wait_deploy` - maybe we should change that and use it properly instead of fixed `--timeout=300s`.

After that change, I think we should be able to change this whole block with:
```suggestion
  wait_deploy scanner-v4-matcher "${SCANNER_V4_MATCHER_READINESS_MAX_WAIT}"
```
if my understanding is correct. As long as we do not have a ready pod, a deployment will not have finished rollout status?